PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,68 @@
+# Model Name
+
+## Use-cases
+Model use cases
+
+## Description
+Description of model
+
+## Model
+
+ |Model        |Download  | Download (with sample test data)|ONNX version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|Model Name       |ONNX Model download link with size  |  tar file containing ONNX model and synthetic test data|ONNX version used for conversion|Accuracy values |
```

Comment:
Should we clarify here that the test data should be in the `.pb` format

Reply:
updated PR template with clarification